### PR TITLE
Linux installation instructions: Add sudo configuration section

### PR DESCRIPTION
**Pull request by pdcastro**
Connects-to: #2149
Change-type: patch

This PR was inspired by #2206 and special thanks go to @frederikheld 🙏

### CODE REVIEW

File: INSTALL-LINUX.md [hunk 1]
```diff
@@ -11,35 +11,52 @@ Selected operating system: **Linux**
...
+If your Linux distribution has an `/etc/sudoers` file that defines a `secure_path`
+setting, run `sudo visudo` to edit it and add the balena CLI's installation folder to
+the ***pre-existing*** `secure_path` setting, for example:
+
+```text
+Defaults   secure_path="/home/james/balena-cli:<pre-existing entries go here>"
+```
+
+If an `/etc/sudoers` file does not exist, or if it does not contain a pre-existing
+`secure_path` setting, do not change it.
```
Comment (pdcastro) [inline INSTALL-LINUX.md:44]: As documented at:
> https://www.sudo.ws/man/1.8.18/sudoers.man.html
> `secure_path`
> Path used for every command run from `sudo`. If you don't trust the people running `sudo` to have a sane `PATH` environment variable you may want to use this. Another use is if you want to have the “root path” be separate from the “user path”. Users in the group specified by the _exempt_group_ option are not affected by _secure_path._ This option is not set by default.

**Comment by ab77** [comment]
@balena-ci retest

**Comment by ab77** [comment]
@balena-ci rebase

**Comment by pdcastro** [comment]
@balena-ci rebase

**Review by srlowe — APPROVED** [review]

**Pull request merged**